packages/snowpack/src/build/build-pipeline.ts

```diff
@@ -91,8 +124,9 @@ async function runPipelineTransformStep(
     }
     for (const destExt of Object.keys(output)) {
       const destBuildFile = output[destExt];
+      const contents = typeof destBuildFile === 'string' ? destBuildFile : destBuildFile.code;
```

Comment:
As a general rule, I like to standardize on a single data format where we can. Can we either:

1. Always store `output[ext] = {code: string, map: string | undefined}`
2.Always store `output[ext] = string`, and then store the source map as a separate entry in the result object.

Reply:
Standardized! Types were simplified. It should all be normalized within `config.ts` now.